## Description: add some description
## Origin/Author: add some origin or author
## Bug: bug URL
Index: acerhk-0.5.35/Makefile
===================================================================
--- acerhk-0.5.35.orig/Makefile	2011-05-16 23:30:09.000000000 +0200
+++ acerhk-0.5.35/Makefile	2011-05-16 23:26:38.000000000 +0200
@@ -14,7 +14,7 @@
 CONFIG_ACERHK?=m
 obj-$(CONFIG_ACERHK) += acerhk.o
 
-EXTRA_CFLAGS+=-c -Wall -Wstrict-prototypes -Wno-trigraphs -O2 -fomit-frame-pointer -fno-strict-aliasing -fno-common -pipe
+EXTRA_CFLAGS+=-c -Wall -Wstrict-prototypes -Wno-trigraphs -O2 -fno-strict-aliasing -fno-common -pipe
 INCLUDE=-I$(KERNELSRC)/include
 
 ifeq ($(KERNELMAJOR), 2.6)
Index: acerhk-0.5.35/acerhk.c
===================================================================
--- acerhk-0.5.35.orig/acerhk.c	2011-05-16 23:30:09.000000000 +0200
+++ acerhk-0.5.35/acerhk.c	2011-05-16 23:28:38.000000000 +0200
@@ -576,7 +576,7 @@
 						 "popa\n\t"
 						 :
 						 :"m" (bios_routine), "m" (buf)
-						 :"%eax", "%ebx", "%ecx", "%edx", "%edi", "%esi", "%ebp"
+						 :"%eax", "%ebx", "%ecx", "%edx", "%edi", "%esi"
 						 );
       local_irq_enable();
   }
@@ -611,7 +611,7 @@
 						 "popa\n\t"
 						 :
 						 :"m" (bios_routine), "m" (preg400), "m" (buf)
-						 :"%eax", "%ebx", "%ecx", "%edx", "%edi", "%esi", "%ebp"
+						 :"%eax", "%ebx", "%ecx", "%edx", "%edi", "%esi"
 						 );
       local_irq_enable();
   }
